.prettierrc.yaml

```diff
@@ -1,2 +1,5 @@
-printWidth: 80
+semi: false
+singleQuote: true
+printWidth: 100
+trailingComma: all
```

Comment:
`trailingComma: all` is Prettier's default, we are discussing this in https://github.com/CheckerNetwork/node/pull/710/files#r2024437202